templates/job_request_detail.html

```diff
@@ -36,6 +35,11 @@ <h1 class="text-3xl tracking-tight break-words font-bold text-slate-900 sm:text-
             Job request: {{ job_request.pk }}
           </h1>
           <dl class="flex flex-col gap-2 text-sm text-slate-600 items-center sm:flex-row sm:gap-x-6 sm:justify-center lg:justify-start">
```

Comment:
Tiny styling tweak:

- Add class `leading-snug` to the `h1`
- Add class `mt-1` to the `dl`

Reply:
`leading-snug` is overridden by `sm:text-4xl` at all widths, is that expected?